### PR TITLE
Updated exposed port to 5601

### PR DESCRIPTION
**Pull request by mihaiparv**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,19 +1,21 @@
 #
 # Kibana Dockerfile
 #
-# https://github.com/eliotk/docker-kibana
+# https://github.com/mihaiparv/docker-kibana
```
Comment (eliotk) [inline Dockerfile]: would you just mind reverting this change for the pull?

**Comment by eliotk** [comment]
Thanks so much for the pull @mihaiparv ! It's much appreciated. I really like how you set the version to env var in the dockerfile. And thanks for catching the port problem. I'm happy to merge this in, if you could just back that url reference change out, it would be appreciated.

Thanks again!

**Comment by mihaiparv** [comment]
Please note I reverted back also the version to install. I have been struggling to get both es and kibana configured as two [fig](http://www.fig.sh/) services as from version 4.0 kibana does not wait for es to get started. Apparently it will get fixed soon with this [pull](https://github.com/elasticsearch/kibana/pull/3033)

**Comment by eliotk** [comment]
thanks again @mihaiparv for this pull and apologies for the lag in merging it in

**Pull request merged**